scripts/generate_default_configs.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python3
+"""
+This script can be used to generate default configuration files for all EuroPiScripts that use the 
+configuration feature. Simply execute this script from the root of the project directory.
+
+The config files will be generated in the a `config` directory. The files can be edited and then
+loaded onto the pico in a `config` directory in the root of the pico's file system.
+"""
+import os
+import sys
+import importlib
+from types import ModuleType
+
+
+def find_europi_scripts():
+    modules = sorted(
+        set(
+            f.partition(".")[0]
+            for f in os.listdir(importlib.import_module("contrib").__path__[0])
+            if f.endswith((".py", ".pyc", ".pyo")) and not f.startswith("__init__.py")
```

Comment:
Mostly because this is general code that I copied from the linked stackoverflow answer. I didn't see a good reason to tailor it to our specific repo, but I can if you think it make the code more confusing. It's going to be fairly deep into the python import mechanics either way.

Reply:
Since it's just importing any of those files it finds, I think it's fine to keep it as is.